client/src/views/Header.tsx

```diff
@@ -34,33 +47,49 @@ const useStyles = makeStyles((theme: Theme) => ({
     justifyContent: 'flex-end',
   },
   navLink: {
-    fontWeight: 'bold',
+    fontWeight: 'normal',
     textDecoration: 'none',
     color: 'black',
-    padding: '0 10px',
+    // padding: '0 10px',
   },
   appBar: {
-    backgroundColor: 'white',
+    backgroundColor: '#ffffff',
     boxShadow: theme.shadows ? theme.shadows[1] : 'none',
   },
   toolbar: {
     padding: '5px 5%',
   },
+  menuItemIconRight: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  bellIcon: {
+    maxWidth: '50px',
+    maxHeight: '50px',
+  },
 }));
 
 function Header() {
   const classes = useStyles();
   const [user, setUser] = React.useContext(UserContext);
   const history = useHistory();
 
-  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const [anchorEl1, setAnchorEl1] = React.useState<null | HTMLElement>(null);
+  const [anchorEl2, setAnchorEl2] = React.useState<null | HTMLElement>(null);
+  const open1 = Boolean(anchorEl1);
+  const open2 = Boolean(anchorEl2);
```

Comment:
for boolean vars, good convention is to prefix with `is`
and let's be more descriptive than `1` or `2`
so maybe `isNavMenuOpen` and `isProfileMenuOpen`
and if you could change the `anchorEl1` state names too, that'd be great